### PR TITLE
Enable test

### PR DESCRIPTION
**Pull request by daniel-butler-irl**
### Description

Tests should be working now, They identified some inconsistencies in the stack_def, the fixes for that are in this PR too. 

### Release required?

- [ ] No release
- [x] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: tests/pr_test.go [hunk 1]
```diff
@@ -2,32 +2,37 @@ package tests
...
+	}
 	options.StackInputs = map[string]interface{}{
-		"resource_group_name": options.ResourceGroup,
-		"ibmcloud_api_key":    options.RequiredEnvironmentVars["TF_VAR_ibmcloud_api_key"],
-		"prefix":              options.Prefix,
+		"resource_group_name":         options.ResourceGroup,
+		"ibmcloud_api_key":            options.RequiredEnvironmentVars["TF_VAR_ibmcloud_api_key"],
+		"prefix":                      options.Prefix,
+		"signing_key":                 privateKey,
+		"secret_manager_service_plan": "standard",
```
Comment (ocofaigh) [inline tests/pr_test.go:35]: ah darn, thats going to be expensive. Every commit to every PR will create a new standard instance, but we dont have trial instance exposed in the fscloud account :/
Not sure what else we can do here unless we ask for quota extension on this account too?

File: tests/pr_test.go [hunk 2]
```diff
@@ -2,32 +2,37 @@ package tests
...
 			"2c - Security Service - Security Compliance Center",
 			"3 - Observability - Logging Monitoring Activity Tracker",
 			"4 - WatsonX SaaS services",
-			"5 - RAG Sample App - Code Engine Toolchain Config",
+			"5 - Sample RAG app - Application Lifecycle Management",
 			"6 - Sample RAG app configuration",
 		},
 	})
 
+	privateKey, _, kerr := common.GenerateTempGPGKeyPairBase64()
```
Comment (ocofaigh) [inline tests/pr_test.go:26]: any GPG key works for this ye?

Comment (daniel-butler-irl) [reply tests/pr_test.go:26]: yes, as long as its valid

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**